src/geo/transform.js

```diff
@@ -295,6 +295,12 @@ class Transform {
      * @returns {number} pixel coordinate
      */
     latY(lat: number) {
+        const validLatitude = 85.05113;
```

Comment:
Declare as static constant outside this method, maybe in `util.js` or on `LngLat`

Reply:
Actually, you have `this.latRange = [-85.05113, 85.05113];` in the same file, as well as couple of `-85.0511` values in style-spec/reference/v8.json.

Reply:
Good catch. I'll refactor these references to a single place.

Reply:
You probably cannot refactor the json one, but maybe it's a good idea to have the same precision level in all 3 places (that is, probably stick with `-85.0511`)

Reply:
I don't think it should be declared in LngLat because the way I understand it is that the LngLat is based around EPSG84 coordinates which are valid up to +-90° so the issue is more in projecting from those coordinates to valid Web Mercator ones. I'm leaning towards using the `latRange` method in Transform.

Reply:
Using `latRange` would prevent general transform/projection calls from using otherwise valid calls with `LatLng` outside the bounds specified in `Map#setMaxBound`. Im not sure that would be clear/expected.

Reply:
Ahhhhh, yeah I was misunderstanding what `latRange` was. Thanks for the clarification.